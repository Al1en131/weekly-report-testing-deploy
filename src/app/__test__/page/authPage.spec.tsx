import AuthPage from "@/app/auth/page";
import LayoutSuperAdmin from "@/app/auth/layout";
import { render } from "@testing-library/react";

describe("Login Page", () => {
    it("render Login page", () => {
        const { container } = render(<AuthPage />);
        expect(container).toMatchSnapshot();
    });
})

describe("Layout Page", () => {
    it("render Layout page", () => {
        const { container } = render(<LayoutSuperAdmin> </LayoutSuperAdmin>);
        expect(container).toMatchSnapshot();
    });
})


