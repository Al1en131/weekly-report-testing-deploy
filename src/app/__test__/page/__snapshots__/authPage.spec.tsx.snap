// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Page render Layout page 1`] = `
<div>
  <main
    class="bg-primary  min-h-screen"
  >
     
  </main>
</div>
`;
